src/shared/components/selected-species/SelectedSpecies.tsx

```diff
@@ -15,75 +15,60 @@
  */
 
 import React from 'react';
-import classNames from 'classnames';
 
-import SelectedSpeciesContent from './SelectedSpeciesContent';
+import SelectedSpeciesLozenge from './SelectedSpeciesLozenge';
```

Comment:
why are we not calling it `SpeciesLozenge` or `CommittedSpeciesLozenge` . `SelectedSpeciesLozenge` sounds like the lozenge that the user clicked on and is selected.

Reply:
> SelectedSpeciesLozenge sounds like the lozenge that the user clicked on and is selected.

We use the word "active" for something that has been clicked on and this is selected (focused on) 🙂 . "Selected species" is the result of selecting a species in species selector. I was debating whether to call is `SpeciesLozenge` or `SelectedSpeciesLozenge`, but thought that `SelectedSpeciesLozenge` is safer — it's a lozenge that represents a selected species. Can change it to `SpeciesLozenge` if people have a strong preference for that name.